lib/local-storage.js

```diff
@@ -21,6 +21,20 @@ function Storage(config) {
   var self = Object.create(Storage.prototype)
   self.config = config
   self.logger = Logger.logger.child({ sub: 'fs' })
+  //TODO -- Figure out if there is some concept of default config?
+  // If a store plugin hasn't been configured, put in the local-fs one.
+  if(self.config.store == undefined) {
+    self.config.store = {'./disk-store': 'disk'}
+  }
+  var store = load_plugins(config, self.config.store, {
+    config : config,
+    logger : self.logger,
+  }, function(p) {
+    return true
+  })
+  // TODO -- support multiple stores? I vote no. Bomb out in this case
```

Comment:
I still don't have head around all of the capabilities of what is/isn't possible. That makes sense though. Public modules are a cache, but the private published modules are records.